h/api/models/test/elastic_test.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+from mock import patch
+from mock import PropertyMock
+
+from h.api.models.elastic import Annotation
+
+
+def test_uri():
+    assert Annotation(uri="http://foo.com").uri == "http://foo.com"
+
+
+def test_uri_with_no_uri():
+    assert Annotation().uri == ""
+
+
+def test_uri_when_uri_is_not_a_string():
+    for uri in (True, None, 23, 23.7, {"foo": False}, [1, 2, 3]):
+        assert isinstance(Annotation(uri=uri).uri, unicode)
+
+
+def test_target_links_from_annotation():
+    annotation = Annotation(target=[{'source': 'target link'}])
+    assert annotation.target_links == ['target link']
+
+
+def test_parent_returns_none_if_no_references():
+    annotation = Annotation()
+    assert annotation.parent is None
+
+
+def test_parent_returns_none_if_empty_references():
+    annotation = Annotation(references=[])
+    assert annotation.parent is None
+
+
+def test_parent_returns_none_if_references_not_list():
+    annotation = Annotation(references={'foo': 'bar'})
+    assert annotation.parent is None
+
+
+@patch.object(Annotation, 'fetch', spec=True)
+def test_parent_fetches_thread_parent(fetch):
+    annotation = Annotation(references=['abc123', 'def456'])
+    annotation.parent
+    fetch.assert_called_with('def456')
+
+
+@patch.object(Annotation, 'fetch', spec=True)
+def test_parent_returns_thread_parent(fetch):
+    annotation = Annotation(references=['abc123', 'def456'])
+    parent = annotation.parent
+    assert parent == fetch.return_value
+
+
+@pytest.fixture
+def link_text(request):
```

Comment:
What triggers the use of these fixtures? I expected to see references to `link_text` as arguments to the `test_` functions.